inc/stat.class.php

```diff
@@ -1526,7 +1526,7 @@ static function title() {
             }
             if (is_array($pages) && count($pages)) {
                foreach ($pages as $page => $name) {
-                  $names[$plug.'/'.$page] = ["name" => $name,
+                  $names[Plugin::getWebDir($plug, false).'/'.$page] = ["name" => $name,
```

Comment:
I do not understand why you use param `$full = false` here to preprend `$CFG_GLPI["root_doc"]` on next loop.